certified-connectors/Eigen Events Connector/apiDefinition.swagger.json

```diff
@@ -0,0 +1,298 @@
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Response to successfully created webhook.",
+            "schema": {
+              "$ref": "#/definitions/WebhookCreationResponse"
+            }
+          },
+          "default": {
+            "description": "Response to failed request.",
+            "x-ms-summary": "Failure response"
+          }
+        }
+      }
+    },
+    "/events/api/webhook/delete/{webhookId}": {
+      "delete": {
+        "description": "This action deletes a webhook you had created",
+        "summary": "Delete a webhook you created",
+        "operationId": "DeleteWebhook",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "webhookId",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "description": "The ID of the webhook to delete",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Response to successfully deleting a webhook."
+          },
+          "default": {
+            "description": "Response to failed request.",
+            "x-ms-summary": "Failure response"
+          }
+        }
+      }
+    },
+    "/events/api/events": {
+      "get": {
+        "description": "This action will fetch all events supported by this server",
+        "summary": "Get Events Available",
+        "operationId": "GetSupportedEvents",
+        "responses": {
+          "200": {
+            "description": "Response for events",
+            "x-ms-summary": "Success response",
+            "schema": {
+              "$ref": "#/definitions/SupportedEventsBody"
+            }
+          },
+          "default": {
+            "description": "Failed request.",
+            "x-ms-summary": "Failure response"
+          }
+        }
+      }
+    },
+    "/events/api/schema/{eventIds}": {
+      "get": {
+        "description": "This action will fetch the schema for the selected eventId",
+        "summary": "Get Dynamic Schema For Event",
+        "operationId": "GetDynamicEventSchema",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "eventIds",
+            "in": "path",
+            "required": true,
+            "description": "Get the schema for event",
+            "type": "number",
+            "x-ms-url-encoding": "single",
+            "default": 1
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response for Schema",
+            "x-ms-summary": "Schema response",
+            "schema": {
+              "$ref": "#/definitions/DynamicSchemaForEvent"
+            }
+          },
+          "default": {
```

Comment:
@tonyscofieldkim Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Refer - [Documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-responses)